charts/edge/templates/NOTES.txt

```diff
@@ -1,7 +1,8 @@
-{{- if not .Values.mezmoApiAccessToken -}}
-#####################################################################
-###  ERROR: Please specify an access token `mezmoApiAccessToken`  ###
-#####################################################################
+{{- if and (not .Values.mezmoApiAccessToken) (not .Values.mezmoApiAccessSecret) -}}
+######################################################################
+###  ERROR: Please specify an access token (`mezmoApiAccessToken`) ###
```

Comment:
`Error` makes sense since it's functionally required to operate, though not install. Updated the flow to block installation instead of just showing the error